config/settings.yml

```diff
@@ -243,6 +243,19 @@ defaults: &defaults
       password: <%= cloudProvider.password %>
       strictSSL: false
       container: <%= cloudProvider.containerPrefix %>-broker
+  virtual_host:
```

Comment:
Virtual host manager needs to store relationship between virtual host instance id and the deployment name on which it was created. 
They could be stored in any container. We discussed with Jagadish and decided that instead of storing it inside the container used by service fabrik we will use a separate container and hence
separate setting for provider of virtual host will be needed.

Reply:
Please change the container name in the settings then. Also I would imagine the other values of provider would be same. Please try to reuse them in that case instead of copying them again.

Reply:
We are planning to reuse the provider container speicifc for rabbitmq service that already exists in landscapes. Other services use it for backup metadata purposes but because for RabbitMQ there is no such use case we will reuse it to store virtual host instances metadata.
Credentials for this container are different than the credentials for service-fabrik broker container. 
In settings.yml I will add a separate section for virtual host cloud provider settings and will use it to at virtual_host.provider.
Is it fine?

Reply:
Yes, that will be better then.